package.json

```diff
@@ -319,6 +325,11 @@
           "command": "rtlDebugger.unWatchVariable",
           "when": "view == rtlDebugger.sidebar && viewItem =~ /inWatchList/",
           "group": "inline"
+        },
+        {
+          "command": "rtlDebugger.addToWaveform",
+          "when": "view == rtlDebugger.sidebar && viewItem =~ /variable/",
```

Comment:
This is wrong, it should check if the waveform viewer is active but I can't figure out how